pydanfossair/danfossclient.py

```diff
@@ -38,8 +38,8 @@ def _command(self, command, socket):
 
         if isinstance(command, UpdateCommand):
             return self._update_command(command, socket)
-
-        raise Exception("Not yet implemented")
+        else:
+            raise Exception("Not yet implemented")
```

Comment:
You should wither keep the raise Exception statement outside else or use 
```
if
elif
else
```

construction.

Reply:
Yes correct, fixed.